include/z64actor.h

```diff
@@ -454,14 +468,14 @@ typedef enum {
     /* 3 */ DOORLOCK_MAX
 } DoorLockType;
 
-// Targetability / ACTOR_FLAG_TARGETABLE?
-#define ACTOR_FLAG_1             (1 << 0)
-// 
+// Allows Tatl to fly over the actor and Z-targeting it
+#define ACTOR_FLAG_TARGETABLE    (1 << 0)
+// Unused
 #define ACTOR_FLAG_2             (1 << 1)
-// 
-#define ACTOR_FLAG_4             (1 << 2)
-// 
-#define ACTOR_FLAG_8             (1 << 3)
+// Changes the targeting behaviour for unfriendly actors (sound effects, Player's stance, etc)
+#define ACTOR_FLAG_UNFRIENDLY    (1 << 2)
```

Comment:
What's the difference/distinction between `Unfriendly` and `Enemy`?

Reply:
Original conversation:
https://discord.com/channels/688807550715560050/688851317593997489/935571349492105337

Although it's not brought up, perhaps the term `Unfriendly` is used to not group `ACTOR_FLAG_UNFRIENDLY` with `ACTORCAT_ENEMY`?

Reply:
It is for enemies, yes, but it is also used for `ACTORCAT_BOSS`s (obviously), `ACTORCAT_NPC` (EnHintSkb, EnSsh, EnNpc), `ACTORCAT_ITEMACTION` (EnClearTag) and `ACTORCAT_PROP` (EnWf, EnRaf, EnBsb).